### ISSUE TITLE
App signed validation fails with signed pkg

### ISSUE THREAD

**Issue by dtsander** [micromdm/micromdm#968]
### What version of `micromdm` are you using?
v1.11.0 (latest release).

### What `micromdm` command did you run?
```
mdmctl apply app -pkg mypackage-signed.pkg -upload
```

### What did you expect to see?
App upload successful.

### What did you see instead?
```
USAGE
  mdmctl apply app [flags]

FLAGS
  -manifest -        path to an app manifest. optional,
                     will be created if file does not exist.
  -md5size 10485760  md5 hash size in bytes (optional)
  -out               pkg output path for -sign and -sign-identity (optional, defaults to OS temporary directory)
  -password          password used for -sign-identity (optional)
  -pkg               path to a distribution pkg.
  -pkg-url           use custom pkg url
  -sign              sign package before importing, requires specifying id of developer identity in keychain (optional, macOS only)
  -sign-identity     sign package before importing, requires specifying path to .p12 developer identity (optional)
  -upload false      upload package and/or manifest to micromdm repository.

MDM packages must be signed. Provide signed package or -sign or -sign-identity flags
```

---

The pkg is signed correctly for sure, but the MDM validation fails.

**I have created this test script to debug the issue:**
```
package main

import (
	"fmt"
	"os"

	goerrors "errors"

	macospkg "github.com/korylprince/go-macos-pkg"
	"github.com/pkg/errors"
)

func checkSignature(pkgpath string) (bool, error) {
	buf, err := os.ReadFile(pkgpath)
	if err != nil {
		return false, errors.Wrap(err, "reading package")
	}
	if err = macospkg.VerifyPkg(buf); err != nil {
		if goerrors.Is(err, macospkg.ErrNotSigned) {
			return false, nil
		}
		return false, errors.Wrap(err, "verifying package")
	}
	return true, nil
}

func main() {
	signed, err := checkSignature("mypackage-signed.pkg")
	fmt.Println("signed", signed)
	fmt.Println("err", err)
}
```

* With github.com/korylprince/go-macos-pkg@v1.3.5 - This is the version used in v1.11.0 is the output **signed false**
* With github.com/korylprince/go-macos-pkg@v1.3.6 - This is the version used on the main branch is the output **signed true**

Based on this I would expect that the issue is already fixed on the current dev version of MicroMDM, but I couldn't find an issue. Any workaround? When you expect a new release?

**Comment by korylprince** [comment]
There's actually additional fixes waiting for merge here: https://github.com/micromdm/micromdm/pull/962

You can build mdmctl from that branch to have it working now on linux, or you can also use mdmctl on macOS, which falls back using pkgutil.

**Comment by dtsander** [comment]
Ok. Thanks. I have created a custom build of v1.11.0 containing the fix.

**Comment by korylprince** [comment]
This fix was released with v1.12 today.
